net/safe-search/Makefile

```diff
@@ -6,8 +6,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=safe-search
-PKG_VERSION:=2.0.0
-PKG_RELEASE:=3
+PKG_VERSION:=2.0.1
+PKG_RELEASE:=1
```

Comment:
could change this to $(AUTORELEASE)

Reply:
Which is not mandatory, it is up to you.

Reply:
I love it!
I've forgotten so many times to update PKG_RELEASE...